website/src/app/kb/deploy/dns/readme.mdx

```diff
@@ -78,9 +78,9 @@ or CIDR Resource in your account.
 
 <Alert color="info">
 
-Firezone only intercepts queries for the `A`, `AAAA`, and `PTR` record types for
-your DNS Resources. All other record types are forwarded to the upstream
-resolver(s).
+Firezone only intercepts queries for the `A`, `AAAA`, `PTR`, `SRV` and `TXT`
```

Comment:
Should we note that SRV and TXT are forwarded to the respective site?

Reply:
I think it's explained in detail in a couple places already (basically functions the same as other records)